src/modules/integrator-guide/pages/data-privacy/data-privacy-officer.adoc

```diff
@@ -7,7 +7,7 @@ Using the data privacy officer console, a DPO can accept or reject requests, lea
 
 == Data Privacy Officer Console
```

Comment:
Please expand/rework the `Features of Data Privacy Officer Console` and `Capabilities of Data privacy officer(DPO)` sections to provide more details into features available for the DPO. Currently they are sort of thrown on the table -- `Search for records with data-sensitivity fields to collect requested data` - how? `The DPO has access to a list of pending privacy requests that awaits his approval.` - where and how do I do that?